app/controllers/password_changes_controller.rb

```diff
@@ -85,7 +85,7 @@ def update
       end
 
       if @password_change_user.save
-        session[:user_id] ||= @password_change_user.id
+        sign_in(@password_change_user, remember_me: session[:remember_me])
```

Comment:
Made a change in #6510 which means we can just do:

```suggestion
        sign_in(@password_change_user)
```